cli_tools_tests/e2e/gce_windows_upgrade/test_suites/windows_upgrade/windows_upgrade_test_suite.go

```diff
@@ -134,6 +149,55 @@ func runWindowsUpgradeNormalCase(ctx context.Context, testCase *junitxml.TestCas
 		true, false, "", false, 0, false)
 }
 
+func runWindowsUpgradeStaging2012(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+
+	runWindowsUpgradeStaging(ctx, testCase, logger,
+		testProjectConfig, testType, standardImage, "windows-2008r2", "windows-2012r2")
+}
+
+func runWindowsUpgradeStaging2016(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+
+	runWindowsUpgradeStaging(ctx, testCase, logger,
+		testProjectConfig, testType, standardImage2012, "windows-2012r2", "windows-2016")
+}
+
+func runWindowsUpgradeStaging2019(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+
+	runWindowsUpgradeStaging(ctx, testCase, logger,
+		testProjectConfig, testType, standardImage2012, "windows-2012r2", "windows-2019")
+}
+
+func runWindowsUpgradeStaging2022(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+
+	runWindowsUpgradeStaging(ctx, testCase, logger,
+		testProjectConfig, testType, standardImage2012, "windows-2012r2", "windows-2022")
+}
+
+func runWindowsUpgradeStaging(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType, testImage string, sourceOS string, targetOS string) {
+
+	suffix := path.RandString(5)
+	instanceName := fmt.Sprintf("test-upgrade-staging-%v", suffix)
+	instance := fmt.Sprintf("projects/%v/zones/%v/instances/%v",
+		testProjectConfig.TestProjectID, testProjectConfig.TestZone, instanceName)
+
+	argsMap := map[e2e.CLITestType][]string{
+		e2e.Wrapper: {
+			"-client-id=e2e",
+			fmt.Sprintf("-source-os=%v", sourceOS),
+			fmt.Sprintf("-target-os=%v", targetOS),
+			fmt.Sprintf("-instance=%v", instance),
+			fmt.Sprintf("-use-staging-install-media=%v", false),
```

Comment:
Here as well, if the `-use-staging-install-media` value is always the same you don't need to format it.